src/PostgresSQL

```diff
@@ -0,0 +1,21 @@
+create table "person" (
+	id INT not NULL UNIQUE,
```

Comment:
Small note: if `NULL` in uppercase, than `not` should be too

Reply:
Thank you, I got it)